CAM-14186: doc(run): add new run features in update docs (#1165)
camunda/camunda-docs-manual: content/update/minor/716-to-717/_index.md

in the [LDAP Adminstrator Authorization section]({{< ref "/user-guide/camunda-bpm-run.md#ldap-administrator-authorization" >}})
of the Camunda Run documentation. You can also find a template LDAP configuration in the Camunda Run `production.yml`.

## Groovy scripting engine included

The Camunda Run distribution now ships with the Groovy scripting engine as well, in the form of the 
`groovy-all-2.4.13.jar` located in the `{RUN_HOME}/configuration/userlib/` directory. Users that utilize Groovy scripts

> in their BPMN processes don't need to provide a Groovy `.jar` archive when upgrading to a new Camunda Run version 

Reviewer: `jar` archive seems a little odd since the `ar` in `jar` already means archive. I would propose to use the word `binary` instead.
```suggestion
in their BPMN processes don't need to provide a Groovy binary when upgrading to a new Camunda Run version 
```

anymore.

Users that don't use Groovy can just remove the `groovy-all-2.4.13.jar` archive from the 
`{RUN_HOME}/configuration/userlib/` directory.

## Camunda Connect plugin included

With Camunda Run 7.17.0, the Camunda Connect process engine plugin is available in Camunda Run just like with any 

> other Camunda distro. Users that use connectors in their BPMN processes don't need to provide any additional `.jar` 

Reviewer: Same as above
```suggestion
other Camunda distro. Users that use connectors in their BPMN processes don't need to provide any additional binary
```

> archive when upgrading to a new Camunda Run version anymore.

Reviewer: ```suggestion
when upgrading to a new Camunda Run version anymore.
```
